mmv1/third_party/terraform/tests/resource_datastream_connection_profile_test.go

```diff
@@ -0,0 +1,41 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatastreamConnectionProfile_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDatastreamConnectionProfileDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDatastreamConnectionProfile_datastreamConnectionProfileBasicExample(context),
+			},
+			{
+				ResourceName:            "google_datastream_connection_profile.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"connection_profile_id", "location"},
+			},
+			{
+				Config: testAccDatastreamConnectionProfile_datastreamConnectionProfileFullExample(context),
```

Comment:
I don't think this test actually tests the update, it tests replace instead. `password` within `forward_ssh_connectivity` is marked as ForceNew so it triggers a recreation I believe?

Reply:
yep, good call.